src/lib.rs

```diff
@@ -25,6 +25,19 @@
 //! // Same as `page_or_print()`.
 //! "very long text".output_paged();
 //! ```
+//!
+//! # Windows notes
+//!
+//! The default pager on Windows is set to `more.com`.
+//! Unfortunately at the time of writing, more is legacy software which
+//! does not understand multi-byte characters. This leads to mojibake.
+//!
+//! Therefore, when using this crate on Windows, it is advisable to restrict
+//! what you print to the ASCII character set.
+//!
+//! Possible alternative solutions include:
+//! * reading the active console codepage and converting the output
+//! * setting the console codepage, if this is appropriate for your use case
```

Comment:
Sorry to be a bit pedantic here, but could you:

- Capitalize the title `Windows Notes`.
- Reflow the text so as to limit line length at 72 chars.
- Put backticks are `more` in `more is legacy software`.
- Use `-` for the lists, and start the sentences with a capital letter and end with a period.

```suggestion
//! # Windows Notes
//!
//! The default pager on Windows is set to `more.com`. Unfortunately at
//! the time of writing, `more.com` is legacy software which does not
//! understand multi-byte characters. This leads to mojibake.
//!
//! Therefore, when using this crate on Windows, it is advisable to
//! restrict what you print to the ASCII character set.
//!
//! Possible alternative solutions include:
//! - Reading the active console codepage and converting the output.
//! - Setting the console codepage, if this is appropriate.
```